oauthenticator/oauth2.py

```diff
@@ -268,7 +303,7 @@ def login_url(self, base_url):
 
     login_handler = "Specify login handler class in subclass"
```

Comment:
As noted by @NickolausDS this overrides the earlier statement setting the login_handler.